Database/Stored Procedures/add_or_update_podium.sql

```diff
@@ -0,0 +1,93 @@
+/*==============================================================*/
+/* DBMS name:		FestiBase									*/
+/* PDM version:		6											*/
+/* Last edited:		20-12-2017									*/
+/* Edited by:		Mariusz Blautzik							*/
+/* Procedure:		Insert + Update PODIUM						*/
+/*==============================================================*/
+
+DROP PROCEDURE IF EXISTS sp_add_or_update_podium
+GO
+CREATE PROCEDURE sp_add_or_update_podium
+   @insert				 BIT,
+   @podium_number		 INT = NULL,
+   @festival_number      INT,
+   @tent_number			 INT,
+   @name				 VARCHAR(50),
+   @construction_width	 INT,
+   @construction_length	 INT,
+   @construction_height	 INT,
+   @floor_height		 INT,
+   @capacity			 INT,
+   @floor_load			 INT,
+   @free_span_width		 INT,
+   @free_span_length	 INT,
+   @free_span_height	 INT,
+   @environment			 TEXT
+
+AS
+BEGIN
+	BEGIN TRY
+
+		IF (@insert = 1)
+		BEGIN
+			INSERT INTO PODIUM(festival_number, tent_number, "name", construction_width, construction_length, construction_height, 
+			                   floor_height, capacity, floor_load, free_span_width, free_span_length, free_span_height, environment)
+			VALUES( @festival_number,
+					@tent_number,
+					@name,
+					@construction_width,
+					@construction_length,
+					@construction_height,
+					@floor_height,
+					@capacity,
+					@floor_load,
+					@free_span_width,
+					@free_span_length,
+					@free_span_height,
+					@environment)
+
+		END
+	ELSE 
+			BEGIN
+				IF (@podium_number is null)
```

Comment:
IF (@podium_number IS NULL OR @podium_number = 0)

- Zie ook nieuwe template
- Hoofdletters